chili-epfl/FROG
# Login modal

This PR introduces a login form in FROG to allow the user to login with their email and password. Furthermore, it renders the correct information on the TopBar of Single Activity depending on whether the user is logged in or logged out. The ability to switch between logout and sign up has also been implemented. I have moved the call to the server out of Sign Up and Login and into `AccountModal`. This makes SignUp and Login dumb components and they can be tested using storybook in the future. 

**Testing instructions**

- Click login and try to login with an email and password combination that you created. 
- If you don't fill either email or password you should get an alert. 
- Appropriate errors will be displayed as an alert (in the future a toast notification) 
- Once you are logged in, your display name will be displayed beside a logout button.
